frontend/packages/.eslintrc.js

```diff
@@ -9,5 +9,6 @@ module.exports = {
         message: 'Use lodash instead. webpack is configured to use lodash-es automatically.',
       },
     ],
+    'no-console': 2,
```

Comment:
nit: I'd prefer "error" to `2` personally.

Reply:
Please move `no-console: 'error'` to `airbnb-base-overrides.js`